src/App.tsx

```diff
@@ -70,42 +81,27 @@ export class App extends React.Component<{}, State> {
                 }
               >
                 <td>
-                  {good === selectedGood ? (
-                    <button
-                      data-cy="RemoveButton"
-                      type="button"
-                      className="button is-info"
-                      onClick={this.clearHandler}
-                    >
-                      -
-                    </button>
-                  ) : (
-                    <button
-                      data-cy="AddButton"
-                      type="button"
-                      className="button"
-                      onClick={() => {
-                        this.addHandler(good);
-                      }}
-                    >
-                      +
-                    </button>
-                  )}
+                  {good === selectedGood
+                    ? removeButton
+                    : (
+                      <button
+                        data-cy="AddButton"
+                        type="button"
+                        className="button"
+                        onClick={() => {
+                          this.addHandler(good);
+                        }}
+                      >
+                        +
+                      </button>
+                    )}
```

Comment:
that's funny - you have two almost identical buttons
and you've assigned one button into a variable,
but the second remained unchanged - NOT NEAT.

be consistent and treat similar elements in a same manner.

I'd prefer you go with a `renderButton` helper function.

Reply:
see this comment : https://github.com/mate-academy/react_goods-selector/pull/1184#discussion_r1061410101